giraffe/webpack.common.config.js

```diff
@@ -57,31 +57,14 @@ module.exports = {
       {
         test: /\.css$/,
         exclude: /leaflet\.css/,
-        use: [
-          {loader: 'style-loader'},
-          {loader: 'css-loader'},
-        ],
+        use: [{loader: 'style-loader'}, {loader: 'css-loader'}],
       },
       {
         test: /leaflet\.css/,
-        use: [
-          {loader: 'style-loader'},
-          'css-loader',
-        ],
-      },
-      {
-        test: /\.(png|svg|jpg|gif)$/,
-        use: [
-          {
-            loader: 'file-loader',
-            options: {
-              name: '[contenthash:10].[ext]',
-            },
-          },
-        ],
+        use: [{loader: 'style-loader'}, 'css-loader'],
       },
       {
-        test: /\.(woff|woff2|eot|ttf|otf)$/,
+        test: /\.(png|svg|jpg|gif|eot|ttf|woff|woff2|otf)$/,
```

Comment:
No real changes here. Prettier did its thing and I also consolidated two rules into one because they were doing the same thing.